index.html

```diff
@@ -0,0 +1,156 @@
+      <h2>Multi-post-stories</h2>
+      <div class="languages">
+        <p>CANOPY</p>
+        <img src="Counter.png" alt="">
+        <p>Back End Dev</p>
+        <img src="Counter.png" alt="">
+        <p>2015</p>
+      </div>
+      <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="info">
+        <span>html</span>
+        <span>css</span>
+        <span>javascript</span>
+      </div>
+      <button class="btn" type="button">see project</button>
+    </div>
+
+    <div class="project-box">
+        <img src="images/Snapshoot Portfolio (1).png" class="demo">
+          <h2>Tonic</h2>
+          <div class="languages">
+            <p>CANOPY</p>
+            <img src="Counter.png" alt="">
+            <p>Back End Dev</p>
+            <img src="Counter.png" alt="">
+            <p>2015</p>
+          </div>
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="info">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <button class="btn" type="button">see project</button>
+        </div>
+
+        <div class="project-box">
+            <img src="images/Snapshoot Portfolio.png" class="demo">
+              <h2>Multi-post-stories</h2>
+              <div class="languages">
+                <p>CANOPY</p>
+                <img src="Counter.png" alt="">
+                <p>Back End Dev</p>
+                <img src="Counter.png" alt="">
+                <p>2015</p>
+              </div>
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="info">
+                <span>html</span>
+                <span>css</span>
+                <span>javascript</span>
+              </div>
+              <button class="btn" type="button">see project</button>
+            </div>
+</section>
+
+<section class="about">
+    <div class="box-container">
+        <div class="box">
+            <div class="text">
+            <h2>about <br> myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <img src="Connect.svg" class="Connect">
+            <div class="link">
+              <img src="Twitter.svg" class="media"/>
+              <img src="Linkedin.svg"class="media"/>
+              <img src="Vector-1.svg"class="media"/>
+              <img src="github.svg"class="media"/>
+              <img src="Vector.svg"class="media"/>
+            </div>
+            <button class="btn-1" type="button">Get my resume</button>
+        </div>
+        <div class="box">
```

Comment:
- I think it's better to use Flexbox to align the content of the "box" instead of just for "tools". Your portfolio must use the same structure for the same group of information.

Reply:
@ElsonOtake  thanks